content/departments/product-engineering/engineering/process/deployments/playbooks.md

```diff
@@ -46,17 +47,19 @@ Deploys on sourcegraph.com are currently [handled by Renovate](#renovate). The [
+Note: sample for `2021-08-19` code freeze.
 
-During the code freeze, [Renovate](#renovate) will be disabled on **2021-08-18 12:00+00:00** and no automatic updates to Kubernetes manifests will be made. To deploy your changes, you can manually create and merge a PR that updates the Docker image tags in [deploy-sourcegraph-cloud](https://github.com/sourcegraph/deploy-sourcegraph-cloud). You can find the desired Docker image tags by looking at the output of the Docker build step in [CI on sourcegraph/sourcegraph `release/2021-08-19` branch](https://buildkite.com/sourcegraph/sourcegraph/builds?branch=release%2F2021-08-19) or by looking at [Docker Hub](https://hub.docker.com/u/sourcegraph/).
+To ensure stability during a [code freeze](https://en.wikipedia.org/wiki/Freeze_%28software_engineering%29), a separate `release/YYYY-MM-dd` branch will be created from `main`, with only approved commits to be `cherry-picked` onto this [branch](https://github.com/sourcegraph/sourcegraph/tree/release/2021-08-19) for release. To ensure any compability between the `main` and `release/YYYY-MM-dd` branches, **ALL** commits must first be merged to `main` and pass [CI](https://buildkite.com/sourcegraph/sourcegraph/builds?branch=main) for being `cherry-picked`. All tests will be run on the `release/YYYY-MM-dd` branch and must pass before docker images are published to docker hub.
+
+During the code freeze, [Renovate](#renovate) will be disabled on **YYYY-MM-dd 12:00+00:00** and no automatic updates to Kubernetes manifests will be made. To deploy your changes, you can manually create and merge (requires approval from either [CloudDevops](../../cloud/devops/index.md) or [CloudSaaS](../../cloud/saas/index.md)) a PR that updates the Docker image tags in [deploy-sourcegraph-cloud](https://github.com/sourcegraph/deploy-sourcegraph-cloud). You can find the desired Docker image tags by looking at the output of the Docker build step in [CI on sourcegraph/sourcegraph `release/YYYY-MM-dd` branch](https://buildkite.com/sourcegraph/sourcegraph/builds?branch=release%2F2021-08-19) or by looking at [Docker Hub](https://hub.docker.com/u/sourcegraph/).
 
 Once your PR has been merged, you can follow the deployment via [CI on the `release` branch](https://buildkite.com/sourcegraph/deploy-sourcegraph-cloud/builds?branch=release).
 
 ### Manually deploying a service to sourcegraph.com
 
-Sometimes you need to manually deploy a service to sourcegaph.com instead of relying on our CD process.
+Sometimes you need to manually deploy a service to sourcegraph.com instead of relying on our CD process i.e. hotfix during code freeze.
```

Comment:
My comment is addressed to line 66, can you please update it as well?

I think it is worth noticing that in order to get a build with new Docker images one should create a branch with `main-dry-run/` prefix (it enables CI pipeline similar to the pipeline which is run against every commit in `main` branch). Any other branch will have only a faster CI pipeline without many stages including building the images.


Also it may be worth adding a link to https://docs.sourcegraph.com/dev/background-information/continuous_integration#run-types just for further info about CI run types

Reply:
Thanks @sashaostrikov ! Very important thing, I will add this as you suggested 👍